requirements.txt

```diff
@@ -1,9 +1,214 @@
-django-appconf==1.0.1
-django-ses-backend==0.1.1
-Django==1.10
-django_compressor==2.1
-djmail==0.11.0
-futures==3.0.3
-premailer==2.9.6
-psycopg2==2.6.1
-requests==2.9.1
+anaconda-navigator==1.8.7
```

Comment:
Check that you're working with-in a virtual environment and that this anaconda thing still works. You can alternatively also on one of your colleague's machine. This requirement can't be fulfilled on my end

Reply:
I have removed this package to avoid any conflicts